libr/util/str.c

```diff
@@ -216,7 +216,7 @@ R_API int r_str_binstr2bin(const char *str, ut8 *out, int outlen) {
 // Returns the permissions as in integer given an input in the form of rwx, rx,
 // etc.
 R_API int r_str_rwx(const char *str) {
-	int ret = atoi (str);
```

Comment:
not necessarily, rwx can be represented too as 1 = x, 2 = w and 4 = r, so providing r_str_rwx("3") will return 3 which is R + W